src/java.xml/share/classes/javax/xml/catalog/GroupEntry.java

```diff
@@ -40,6 +40,12 @@
     static final int ATTRIBUTE_DEFFER = 1;
     static final int ATTRIBUTE_RESOLUTION = 2;
 
+    //Indicates whether this is the Catalog instance (vs a Catalog entry)
+    boolean isCatalogInstance = false;
```

Comment:
Can we eliminate this instance variable, by checking whether the type is `Catalog` or not in `resetOnStart()`?

Reply:
Yes, thanks!